pkg/report/template.go

```diff
@@ -102,27 +112,75 @@ func renderTemplates(dibReport Report) error {
 		return err
 	}
 
-	// Generate test.html
-	if err := dibReport.renderTemplate("test", nil); err != nil {
-		return err
+	return nil
+}
+
+// printReportURL return a string representing the path from which we can browse HTML report.
+func printReportURL(dibReport Report) string {
```

Comment:
`print` prefix suggests that the function prints to stdout but it only returns the string